README.md

```diff
@@ -29,39 +28,35 @@ A distinctive feature of AssertiveTS with other assertion libraries is that it l
 - Works with any test runner and framework such as [Jest](/docs/jest-tutorial.md), [Mocha](/docs/mocha-tutorial.md), or Ava
 - Well tested: more than 300 tests!
 
-## Install
-
-```sh
-npm install --save-dev @stackbuilders/assertive-ts
-```
+## Packages
 
-Or:
+For conviniece, this library is split into package grouped within the same namespace:
 
-```sh
-yarn add --dev @stackbuilders/assertive-ts
-```
+- **[assertive-ts/core](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md):** Core functionalities, assertions applicable for any kind of application. This package is required to use the [extension mechanism](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md#extension-mechanism-⚙️) (plugins). This is package is the replacement for the deprecated `stackbuilders/assertive-ts` package.
```

Comment:
```suggestion
- **[assertive-ts/core](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md):** Core functionalities, assertions applicable for any kind of application. This package is required for [extension mechanism](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md#extension-mechanism-⚙️) (plugins). Also, this package replaces the deprecated `stackbuilders/assertive-ts` package.
```

Reply:
This suggestion here may not be worth as the same paragraph is duplicated in `core/readme`, and accepting this change my imply keeping the other one in sync with this one, so, up to you.